packages/virtualtours/src/react-components/pages/Home.tsx

```diff
@@ -5,6 +5,14 @@ class Home extends React.Component {
 
 	componentDidMount() {
 		window.addEventListener('scroll', this.handleScroll);
+		let id = window.location.href.split('/')[3];
+		if (id) id = id.substring(1);
+		const element = document.getElementById(id);
+		if (element) {
+			const scrollDiv = element.offsetTop;
+			window.scrollTo({ top: scrollDiv, behavior: 'smooth' });
+		}
+
```

Comment:
- window.location.href will give back a string such as http://localhost:8000/#consultants

-Will split at each slash. Then at the third slash you will get "#consultants" id from there. 

-Substring cuts off the #, then takes "consultants". ID is now "consultants". 

- Find the element with the matching id in landing page, then get offset and scroll to there.